cmd/commands/commands.go

```diff
@@ -1101,6 +1115,7 @@ func closeChannel(ctx *cli.Context) error {
 		SatPerVbyte:     ctx.Uint64(feeRateFlag),
 		DeliveryAddress: ctx.String("delivery_addr"),
 		MaxFeePerVbyte:  ctx.Uint64("max_fee_rate"),
+		NoWait:          ctx.Bool("coop_close_with_htlcs"),
```

Comment:
if `coop_close_with_htlcs` is true then `NoWait` should be false?

Reply:
not really, no wait needs to be true for the coop close to succeed, thats why I renamed it to a more explanatory name like "coop_close_with_htlcs".

Reply:
Checking why it's introduced 804dabb4ea45aabaf959b728b881f4d807409e48, I think we can just set `NoWait` to true here to avoid adding another flag on the cli.

Reply:
ok done